### PR TITLE
Dashboard improvements

### PR DESCRIPTION
**Pull request by bogdal**
This PR:
- [x] fixes paginator urls
- [x] adds appropriate info message on the product list page
- [x] hides `non-existent` user details
- [x] makes description field optional

### CODE REVIEW

File: saleor/dashboard/templates/dashboard/customer/list.html [hunk 1]
```diff
@@ -57,10 +57,14 @@ <h5 class="card-title black-text">{{ title }}</h5>
                                 <a href="{% url "dashboard:customer-details" pk=customer.pk %}">{{ customer.email }}</a>
                             </td>
                             <td>
-                                {{ default_address.first_name }} {{ default_address.last_name }}
+                                {% if default_address.first_name %}
```
Comment (patrys) [inline saleor/dashboard/templates/dashboard/customer/list.html]: Why not `{% if default_address %}`? It's the address that is `None`.

File: saleor/dashboard/customer/views.py [hunk 2]
```diff
@@ -28,6 +28,8 @@ def customer_list(request):
     customers, paginator = paginate(customers, 30, request.GET.get('page'))
     ctx = {'customers': customers, 'form': form, 'title': title,
            'paginator': paginator}
+    if not request.GET:
+        ctx.update({'default_pagination_params': {'order_status': 'on'}})
```
Comment (patrys) [inline saleor/dashboard/customer/views.py]: Why do we need this? Shouldn't the view behave the same no matter what page you're viewing? By default there are no GET params yet the view is filtered.

File: saleor/dashboard/templates/dashboard/customer/list.html (hunk 1 above)
Comment (bogdal) [inline saleor/dashboard/templates/dashboard/customer/list.html]: :+1:

File: saleor/dashboard/customer/views.py (hunk 2 above)
Comment (bogdal) [inline saleor/dashboard/customer/views.py]: We need this to prepare correct pagination links. By default we send only one parameter `page`, which affects on the search form behaviour.

Comment (patrys) [inline saleor/dashboard/customer/views.py]: I mean we should probably fix the view or the form. `request.GET or None` is what causes different behaviour when page number is given. I propose we always pass `request.GET` even when empty and change the default field value to be consistent.

Comment (bogdal) [inline saleor/dashboard/customer/views.py]: :+1: I will change this.

**Comment by patrys** [comment]
Is this ready for merge?

**Comment by bogdal** [comment]
@patrys yes, please merge.

**Pull request merged**